src/core/public/apm_system.ts

```diff
@@ -29,6 +29,8 @@ interface StartDeps {
   application: InternalApplicationStart;
 }
 
+export const CHANGE_ROUTE_TRANSACTION_TYPE = 'change-route';
```

Comment:
Is this type used as a distinction from existing `route-change` transactions from RUM agent? https://github.com/elastic/apm-agent-rum-js/blob/f141c0f504ef8ea2e703d32239344e309f5e38d6/packages/rum-core/src/common/constants.js#L66

Reply:
I think it's the same.
Maybe export the constants from the package, so we can reuse them?

Reply:
Just a typo!

p.s. could apm-rum maybe export those constants?

Reply:
Actually, looking at the data - at the moment two types of events get mixed up under the same name.
![image](https://user-images.githubusercontent.com/3016806/143857730-d4ebdc35-fdb2-4410-9428-8082845560e0.png)

Anything prefixed with `\app\` is logged from `kibana` and anything without it is logged by `apm-rum`. I'm going to rename this event into `app-change` so we can decide if we actually need to track both.

Reply:
Sounds good 👍🏽 

> p.s. could apm-rum maybe export those constants?

Sure we can do that as well.